src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const tableHeader = document.querySelector('thead');
+const tableBody = document.querySelector('tbody');
+const tableRow = [...tableBody.querySelectorAll('tr')];
+
+const moneyToNumber = function(str) {
+  const withoutDollar = str.split('$').join('');
+
+  return +withoutDollar.split(',').join('');
+};
```

Comment:
```suggestion
const moneyToNumber = (str) => {
  const withoutDollar = str.split('$').join('');

  return +withoutDollar.split(',').join('');
};
```

There's nothing wrong with your variant of function expression. It's just a recommendation as a lot of people use either function declartion or arrow functions.